introduction.md

```diff
@@ -1,2 +1,6 @@
 Hi! I am Svetlana Lukyanenko. I am new to dojo
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?